src/classes/LineItemTriggerHandler0088.cls

```diff
@@ -0,0 +1,131 @@
+public with sharing class LineItemTriggerHandler0088 {
+		
+  
+
+	
+	/** 
+	This handler methods is to be used on all relevant triggers events.
+		It updates purchased amount on orders and update top product categories.
+		Logic is fully bulkified. */		
+	public static void handle(List<Line_Item__c> items) {
```

Comment:
### Strong Testing: Untested methods
#### Finding
Method \`handle\` is not tested. Please ensure a test method is added.